pkg/kubectl/cmd/create/create_job_test.go

```diff
@@ -184,6 +185,7 @@ func TestCreateJobFromCronJob(t *testing.T) {
 				Name: jobName,
 			}
 			job := o.createJobFromCronJob(tc.from)
+			tc.expected.OwnerReferences = []metav1.OwnerReference{*metav1.NewControllerRef(tc.from, apps.SchemeGroupVersion.WithKind("CronJob"))}
```

Comment:
Set the expected object as it should look like in appropriate test cases.

Reply:
TBH I'm having a hard time doing that without modifying the whole test case (unless I'm missing something). The test cases are a map of cronjob / job and I'm not aware of a way I can easily reference the cronjob from the job. I've extracted the cronjob outside of the map of test cases, but I can't say I like this more :)